classes/class-sensei-teacher.php

```diff
@@ -1405,4 +1409,41 @@ public function limit_teacher_edit_screen_post_types( $wp_query ) {
 
     } // end limit_teacher_edit_screen_post_types()
 
+
+    /**
+     * Sensei_Teacher::teacher_login_redirect
+     *
+     * Redirect teachers to /wp-admin/ after login
+     *
+     * @since 1.8.7
+     * @access public
+     * @param string $user_login
+     * @param object $user
+     * @return void
+     */
+
+    public function teacher_login_redirect( $user_login, $user  ) {
+
+        var_dump($_POST['redirect']);
+
+        if ( isset( $user->roles ) && is_array( $user->roles ) ) {
```

Comment:
@mahangu I think this check is redundant. We only need to check for the edit_courses below.